typedargs/basic_structures.py

```diff
@@ -4,6 +4,13 @@
 class ParameterInfo:
 
     def __init__(self, type_class, type_name, validators, desc):
+        """
```

Comment:
This description should move to the class docstring and the type details should become proper annotations.

In many places in CoreTools we still put arguments in the docstring only because we used to support python 2 until a few months ago but we no longer do so now we should aggressively convert to using function annotations for this kind of information.

`typedargs` while separate from CoreTools also moved to be python3 only at the same time.

Reply:
done f5acf1206db6